game_eggs/steamcmd_servers/enshrouded/egg-enshrouded.json

```diff
@@ -4,65 +4,45 @@
         "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2024-01-24T16:49:35+01:00",
-    "name": "Enshrouded",
+    "exported_at": "2024-01-26T01:31:21-05:00",
+    "name": "Enshrouded Proton",
     "author": "vapokrocks@gmail.com",
-    "description": "Enshrouded is a survival game currently in beta",
+    "description": "Enshrouded is a game of survival, crafting, and Action RPG combat, set within a sprawling voxel-based continent. As you journey across the mountains and deserts of an open world, you are free to choose your path and shape your destiny.  Credit to gOOvER for the help with the proton image which seems to be performing better over the wine egg.",
     "features": [
         "steam_disk_space"
     ],
     "docker_images": {
-        "Wine Latest": "ghcr.io\/parkervcp\/yolks:wine_latest"
+        "ghcr.io\/goover\/steam:proton": "ghcr.io\/goover\/steam:proton"
```

Comment:
Thats not allowed. That is not an offical image. We have one. If it needs chabged. PR it.

Reply:
Hey I'm not a docker fan and wasn't aware there's a requirement to use parkers images.  His proton image is out of date and doesn't run SteamCMD correctly.

All I'm looking for is what works best for the Enshrouded community.  Not to join my an elitist establishment.

I thought this was a community repo.  But sounds like it's more closed than I expected.

Sad.

Reply:
Please make a PR to my Proton image. I want it to be as good as it can be and any changes for it to be better are a bonus for everyone.

See here for the Dockerfile https://github.com/parkervcp/yolks/tree/master/steamcmd/proton

As part of the contributing I ask that it uses the pterodactyl yolk/images or my own See Number 5 here https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md#general-rules-for-eggs